File: src/hbv/api/dto/user/UserDTO.java
package hbv.api.dto.user;

import java.util.ArrayList;

public class UserDTO
{
    public int id;

    public String username;

    public String imageUrl;

    public String imageFilename;

    public String image;

    public ArrayList<FollowerDTO> followers;

    public ArrayList<FollowDTO> follows;
}

File: app/static/js/dto/user/UserDTO.js
class UserDTO
{
    /**
     * @type {int}
     */
    id;

    /**
     * @type {String}
     */
    userName;

    /**
     * @type {String}
     */
    imageUrl;

    /**
     * @type {String}
     */
    imageFilename;

    /**
     * @type {String} // TODO: Declare as base64
     */
    image;

    /**
     * @type {FollowerDTO[]}
     */
    followers;

    /**
     * @type {FollowDTO[]}
     */
    follows;
}